x-pack/plugins/session_view/public/components/process_tree/helpers.ts

```diff
@@ -32,7 +32,7 @@ export const updateAlertEventStatus = (
         alert: {
           ...event.kibana.alert,
           workflow_status:
-            updatedAlertsStatus[event.kibana.alert?.uuid]?.status ??
+            updatedAlertsStatus[event.kibana?.alert?.uuid ?? '']?.status ??
             event.kibana.alert?.workflow_status,
```

Comment:
does it need a check another on `?? event.kibana.alert?.workflow_status` as well?

Reply:
I think it's fine for `event.kibana.alert?.workflow_status` to be undefined, since we are treating all fields in `event.kibana.alert` to be optional in `ProcessTreeAlert`